### ISSUE TITLE
Config help link is invalid

### ISSUE THREAD

**Issue by Sea-n** [ooni/probe-cli#75]
The link https://ooni.io/help/probe-cli cannot be found.

```json
{
  "_": "This is your OONI Probe config file. See https://ooni.io/help/probe-cli for help",
}
```

Ref: [config.json](https://github.com/ooni/probe-cli/blob/master/data/default-config.json#L2)

**Comment by hellais** [comment]
Thanks for reporting this. You are right that the page in question does not exist yet. Perhaps we should just drop the link while it doesn't exist.

**Comment by bassosimone** [comment]
@hellais yes, I believe we should do that. Also, there's the risk that the page would become stale because we forget about updating it, so maybe the link is not necessarily best to put in there.
